src/main/resources/templates/c_plus_plus/test/testUtils/Tester.py

```
@@ -0,0 +1,81 @@
+            test.case.testerOutput = self.name + "\n" + getTesterOutput()
+            # Update test results:
+            testResults[name] = test.case.result
+        self.__printResult()
+
+    def addTest(self, test: AbstractTest) -> None:
+        """
+        Adds a new test that will be run once "run()" is invoked.
+        """
+
+        if test.name in self.tests:
+            raise ValueError(f"Test '{test.name}' already registered. Test names should be unique!")
```

Ruff

Avoid specifying long messages outside the exception class

(TRY003)